docs/concepts/vector.rst

```diff
@@ -0,0 +1,26 @@
+Vector
+##################################
+
+Vector is lightweight and ultra-fast tool for building observability pipelines.
+In the aspects project it's used as a Ralph replacement, but can be used as
```

Comment:
```suggestion
In the Aspects project it can optionally be used as a replacement for Ralph to capture xAPI learner statements in the ClickHouse database, and/or as a way to store raw tracking log statements. It can be used as a general purpose log collector and forwarder.
```